app/views/account_subscriptions/confirm.html.erb

```diff
@@ -61,3 +61,10 @@
     <%= t("account_subscriptions.confirm.unlinked_subscriptions.after") %>
   </p>
 <% end %>
+
+<%= render "govuk_publishing_components/components/details", {
+  title: t("account_subscriptions.confirm.how_we_use_information.title") 
+  } do %>
+    <%= t("account_subscriptions.confirm.how_we_use_information.expanded_html") %>
+  <% end %>
```

Comment:
Total Nitpick, but is this misaligned?
I would have thought Rubocop would have kicked off about that 🤔